examples/borer-circe-migration/src/main/scala/btest/Transcoding.scala

```diff
@@ -0,0 +1,46 @@
+package pl.marconzet
+package btest
+
+import io.bullet.borer.Dom.ArrayElem.Unsized
+import io.bullet.borer.{Cbor, Codec, Dom, Json}
+import io.bullet.borer.derivation.MapBasedCodecs._
+
+object Transcoding extends App {
+  case class Employee1(nick: String, age: Int)
+
+  case class Employee2(nick: String, age: Option[Int])
+
+  implicit val employee1Codec: Codec[Employee1] = deriveCodec[Employee1]
+  implicit val employee2Codec: Codec[Employee2] = deriveCodec[Employee2]
+
+  val value = Employee2("Alice", None)
+
+
+  val dom = Json // could also be `Json` to target JSON
+    .transEncode(value)
+    .transDecode
+    .to[Dom.Element]
+    .value
+
+  val transformer =
+    new Dom.Transformer {
```

Comment:
`Dom` stands for Document Object Model here? 😯

Reply:
As in, I'd say Document Object Model is only limited to XML/HTML... of course a matter of nomenclature, since logically it's a pretty similar thing anyway here

Reply:
It's [Document Object Model](https://sirthias.github.io/borer/borer-core/DOM.html)